x-pack/legacy/plugins/reporting/types.d.ts

```diff
@@ -182,7 +184,7 @@ export interface ConditionalHeadersConditions {
 }
 
 export interface CryptoFactory {
-  decrypt: (headers?: Record<string, string>) => string;
+  decrypt: (headers?: string) => any;
```

Comment:
don't we always decrypt to `string`? 🤔

Reply:
The decrypt function de-serializes the data as well, so it can be any kind of object.